mod/content.php

```diff
@@ -197,7 +197,7 @@ function content_content(&$a, $update = 0) {
 	}
 
 	if($conv) {
-		$myurl = $a->get_baseurl() . '/profile/'. $a->user['nickname'];
+		$myurl = App::get_baseurl() . '/profile/'. $a->user['nickname'];
```

Comment:
I think we should do this in an upcoming pull request - and we should discuss (in the developer forum) which way we want to go.